frontend/src/app/commonComponents/USAGovBanner.jsx

```diff
@@ -1,7 +1,7 @@
 import React from "react";
-import iconDotGov from "../../img/icons/icon-dot-gov.svg";
-import usFlagSmall from "../../img/icons/us_flag_small.png";
-import iconHttps from "../../img/icons/icon-https.svg";
+import iconDotGov from "../../../node_modules/uswds/dist/img/icon-dot-gov.svg";
```

Comment:
⚠️ is there a cleaner way to do this or is this fine?

Reply:
which aspect of this are you referring to that could be cleaner?